docs/src/modules/java/pages/call-another-service.adoc

```diff
@@ -7,7 +7,7 @@ In some cases it is useful to call a component in another service, for example i
 
 Calling other Kalix services in the same project from an Action is done by invoking them over gRPC much like how an external client would. The service is however identified only by the name it has been deployed as, Kalix takes care of routing requests to the service and keeping the data safe by encrypting the connection for us.
 
-In this sample we will make an action that does two sequential calls to the xref:value-entity.adoc[Value Entity Counter] service, deployed with the service name "counter".
+In this sample we will make an action that does two sequential calls to the xref:value-entity.adoc[Value Entity Counter] service, deployed with the service name "counter."
```

Comment:
Interesting.. this is a new one for me. Always learning I guess :)

Reply:
That looks odd. It's a name, not a quotation that should include punctuation inside the quotes. Maybe should be in backticks instead.

Reply:
I realize this discussion is mostly joking around, but if we disagree with any of these grammar rules, we can override the rule via configuration in https://github.com/lightbend/kalix-jvm-sdk/tree/main/styles